Backend/src/__init__.py

```diff
@@ -1,10 +1,34 @@
 from flask import Flask, Response, request
+import requests
+from src.google import SearchGoogle
+import json
 
-def create_app(test_config=None):
+def exception_handler(message):
+    status_code = 400
+    return {"status_code": status_code, "message": message}
+    return Response(str(message), status_code)
+
+def create_app():
     app = Flask(__name__)
+    search = SearchGoogle()
 
     @app.route("/hello")
     def start():
         return "Hello World!"
     
+    @app.route("/getByIsbn", methods=['POST'])
+    def searchByIsbn():
+        payload = request.get_json()
+        attrs = ['keywords', 'title', 'author', 'publisher', 'subject', 'isbn', 'lccn', 'oclc']
```

Comment:
is there a way to take just a search param and search all fields? otherwise the json might look like { keywords: stephen king, title: steven king, author: steven king.... etc

it seems like for the user it's easiest to have a single place to type a search without specifying which key they are looking for